spec/compiler/macro/macro_methods_spec.cr

```diff
@@ -20,6 +20,20 @@ describe "macro methods" do
       end
     end
 
+    describe "symbolize" do
+      it "expands macro with symbolize call on string" do
+        assert_macro "x", "{{x.symbolize}}", [StringLiteral.new("hello")] of ASTNode, ":\"\\\"hello\\\"\""
+      end
+
+      it "expands macro with symbolize call on symbol" do
+        assert_macro "x", "{{x.symbolize}}", [SymbolLiteral.new("hello")] of ASTNode, ":hello"
```

Comment:
@asterite That makes sense. The above condition was because I was having `(:sym).symbolize == :sym`.